bin/functions

```diff
@@ -108,17 +108,44 @@ function install {
             case ${ASDF_INSTALL_VERSION} in
                 zulu*) mv ./* "${ASDF_INSTALL_PATH}" ;;
                 liberica*) mv ./* "${ASDF_INSTALL_PATH}" ;;
-                *) mv Contents/Home/* "${ASDF_INSTALL_PATH}" ;;
+                *) 
+                    mv Contents/Home/* "${ASDF_INSTALL_PATH}" 
+                    macOS_integration
+                    ;;
             esac ;;
         *) mv ./* "${ASDF_INSTALL_PATH}" ;;
     esac
 }
 
+function uninstall {
+    rm -rf "${ASDF_INSTALL_PATH}"
```

Comment:
Could we move this after the removal of the directory in `/Library/Java/JavaVirtualMachines/`?
Otherwise there might be a race condition where there's a broken symbolic link in `/Library/Java/JavaVirtualMachines/${ASDF_INSTALL_VERSION}/Contents/Home`.

Reply:
Done.